src/python/metrics/fuzzer_stats.py

```diff
@@ -500,6 +500,9 @@ def get(self, group_by, group_by_value):
       corpus_size_bytes = coverage_info.quarantine_size_bytes
       corpus_location = coverage_info.quarantine_location
 
+    if corpus_size_units is None or corpus_size_bytes is None:
```

Comment:
Maybe add a comment that you have in description.

Reply:
done!